cms/templates/cms/home_page.html

```diff
@@ -217,31 +165,75 @@ <h3>Why should I enroll?</h3>
-            <p>Take online courses on a schedule that fits your needs, and meet motivated learners who share your interests.</p>
+            <h4>Advance Your Career</h4>
+            <p>Learn online, pass digital proctored exams, and earn a program credential. 
+              Boost your professional profile with an affordable and valuable credential from MIT.</p>
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
               <img src="{% static 'images/blended.png' %}" alt="">
             </div>
 
             <h4>Accelerate Your Master’s</h4>
-            <p>A MicroMasters program is a faster path to a master’s degree from MIT and other universities, and is recognized by employers for providing deep learning in specific career fields.</p>
+            <p>Fast-track your master’s degree at MIT and  many other <a href="/pathways-graduate-programs/">pathway schools</a> around the world.</p>
 
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/cost.png' %}" alt="">
+              <img src="{% static 'images/dome_icon.webp' %}" alt="">
             </div>
-             <h4>Save Money &amp; Time</h4>
-            <p>A MicroMasters program is an affordable option for professionals who can’t take off time to earn a traditional master’s degree.</p>
+             <h4>Be Part of the MIT Community</h4>
+            <p><a href="/benefits/">Become an affiliate member of the MIT Alumni Association</a> and receive special access to professional journals and publications.</p>
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
               <img src="{% static 'images/globe-icon.png' %}" alt="">
             </div>
              <h4>Grow Your Network</h4>
-            <p>After you earn your MicroMasters® program credential, you will become an affiliate of the MIT Alumni Association.{% if benefits_url %} <a href="{{ benefits_url }}">Learn More</a>{% endif %}</p>
+            <p>After you earn your MicroMasters® program credential, build connections through a private group of like-minded professionals.</p>
+          </div>
+        </div>
+      </section>
+
+      <section class="b2b">
+        <div class="container">
+          <div class="content-holder">
+            <h3>For Organizations and Academic Institutions</h3>
+            <p>
+              Equip your workforce and organization with new applicable skills through the MITx MicroMasters program!
+             </p> 
+             <p>
+              We work closely with organizations globally that wish to integrate our programs into their own education 
+              and training portfolios. Flexible pricing options available. 
+             </p>
+             <a class="btn btn-primary text-uppercase px-5 py-2" href="/organizations/">Inquire Today</a>
```

Comment:
```suggestion
             <a class="btn btn-primary text-uppercase px-5 py-2" href="/organizations/">Learn More</a>
```

Now that the /organizations/ page's content is more clear, a "Learn More" button makes more sense.